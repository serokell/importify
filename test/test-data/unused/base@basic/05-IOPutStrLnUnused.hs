module IOPutStrLnUnused where

import System.IO (putStrLn)

main :: IO ()
main = pure ()
