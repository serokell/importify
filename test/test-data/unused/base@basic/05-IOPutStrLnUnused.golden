module IOPutStrLnUnused where


main :: IO ()
main = pure ()
